### ISSUE TITLE
Using jekyll-less on Codeship

### ISSUE THREAD

**Issue by jekyll-mini-blog** [zroger/jekyll-less#12]
The plugin doesn't compile properly becuse of a dependency issue - Codeship gives an error message saying that the plugin also needs theruberrace gem. This can be added to the readme file by asking the user to add: 
gem 'therubyracer', '~> 0.12.2'

To the gemfile and additionally, to add:
gems: ['jekyll-less', 'therubyracer']

To the _config.yml

Please let me know if this issue occurs on other platforms too, if so, I'll add a quick thing to the readme and send in a pull request. Cheers!

**Comment by martynchamberlin** [comment]
Yes, this problem also occurred for me when attempting to set up this plugin locally on macOS. `therubyracer` is definitely a required dependency for this. Adding it to my Gemfile fixed the problem.
